app/models/merchant.rb

```diff
@@ -0,0 +1,21 @@
+class Merchant < ActiveRecord::Base
+  # Validations
+  validates :name, presence: true, uniqueness: true
+  validates :email, presence: true, uniqueness: true, format: {with: /@/}
+  validates :password, presence: true
+  validates :password_confirmation, presence: true
+
+  # Associations
+  has_secure_password
+
+  has_many :products
+  has_many :orders, :through => :order_items
+  has_many :order_items
+
+  # Scopes
+
+  #Merchant.find(1).shipped(true/false) returns all shipped order items
+  def shipped?(boolean)
```

Comment:
If this method is returning a list of order items, then a `?` method is not the appropriate use. The `?` method should return a true or false value.